boards/arm/nucleo_l073rz/nucleo_l073rz.dts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 Ilya Tagunov <tagunil@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32l073.dtsi>
```

Comment:
I couldn't find stm32l073.dtsi file. Is it missing?

Reply:
My bad. I found it.

Reply:
No, it is added by ebb91ac2a39b76b60e2d892974c09248cae8a240.